task/harden-concourse-tasks/scripts/build.sh

```diff
@@ -93,21 +93,21 @@ ua attach --attach-config ua-attach-config.yaml
 apt-get -y -q install \
   usg \
 
-#install nodejs using nvm
-wget -qO- https://raw.githubusercontent.com/nvm-sh/nvm/${NVM_VERSION}/install.sh | bash
-export NVM_DIR="$([ -z "${XDG_CONFIG_HOME-}" ] && printf %s "${HOME}/.nvm" || printf %s "${XDG_CONFIG_HOME}/nvm")"
-[ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh" # This loads nvm
-
-nvm install "${NODE_VERSION}"
-npm install minimist
-npm install semver
+#install nodejs from source
```

Comment:
I have no objection, but why did we need to install Node from source rather than via `nvm`?

Reply:
nvm only installs node per user rather than globally, and when run as a container the user that's booted up is unable to locate node